### ISSUE TITLE
Can't request funds more than once

### ISSUE THREAD

**Issue by madmartian5** [Zilliqa/ceres#7]
In the faucet, I am unable to request funds more than once for the same address. For development purposes, there should be no limit on number of times that you can request zil from faucet. Also, would be good that default amount is say a large number - 100k ZIL 👍

**Comment by madmartian5** [comment]
I got around this by using the docker cli and renaming `faucet-state.json` file. But I think ultimately, the ability to allow multiple requests should be fixed.. All logic is in faucet.js - I can't do a pull request as I don't have access to the github repository.. and the repository would also be for test (not just local).. so maybe better to have a seperate instance for local dev.

**Comment by bb111189** [comment]
hi @madmartian5 

The faucet is a running service and it is possible to programmatically request funds. In fact, for the upcoming testing library, it will support requesting funds from the facuet.

**Comment by madmartian5** [comment]
Yes, but for ceres which is running the faucet locally there is no reason to have limitations on the number of times an address can request. Having a quick look.. it would be easy enough to change faucet.js to simply bypass user registration into `faucet-state.json` - which is where the faucet is looking to decide whether or not to provide more zil.
